Sluggo/View Controllers/MemberListViewController.swift

```diff
@@ -40,10 +44,23 @@ class MemberListViewController: UITableViewController {
         let cell = tableView.dequeueReusableCell(withIdentifier: "MemberCell", for: indexPath) as UITableViewCell
         let member = self.members[indexPath.row]
         cell.textLabel?.text = member.owner.username
+        cell.detailTextLabel?.text = generateMemberDetail?(member) ?? ""
 
         return cell
     }
 
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        guard let generateController = self.generateSegueableController else { return }
+
+        guard let viewController = generateController(identity) else { return }
+
```

Comment:
We may want a way to adjust this more in the sense that if we want to pass the member's id as a parameter, or just pass that somewhere to know which row it was ran on.

Can be done in another PR.